spec/lib/reference_counter_api_spec.rb

```diff
@@ -38,8 +38,8 @@
         status_code: 403,
         content: {
             'description' =>
-            "mwapi error: permissiondenied - You don't have permission to view deleted " \
-            'text or changes between deleted revisions.'
+            "rest-permission-denied-revision - User doesn't have access to the requested " \
+            'revision (708326238).'
```

Comment:
For the toolforge reference-counter HTML endpoint, we build error description messages using what the `w/rest.php/v1/revision/{revid}/html` endpoint returns (https://en.wikipedia.org/w/rest.php/v1/revision/708326238/html is the request we make to retrieve the HTML).